### ISSUE TITLE
Differentiated Usage of Manifest and Manifestation

### ISSUE THREAD

**Issue by prototypo** [w3c/dpub-pwp#42]
Polymorphism in the terms "manifest" (in a computing context) and "manifestation" (in a FRBR context).

Confusion is worse when a word like "manifested" is used. This section has at least declared its context:

> Section 1.1: "A Web Publication is not just a collection of links— the act of publishing involves obtaining resources and organizing them into a publication, which must be “manifested” (in the FRBR [frbr] sense) by having the resources available on a Web server."

It would be better yet to define both terms, and to remove any variations, as with stemming.

It would be helpful for readability to add a note like this:

> The term "manifestation" is used in this section as defined in FRBR; The term "manifest" is used throughout this specification to mean a resource (a file) containing a list of references to other  resources, in the manner of a shipping list, or a list of things  that needed to be downloaded for offline use.

This issue applies equally to https://www.w3.org/TR/pwp-ucr/ which uses two instances of the word "manifestation".

**Comment by liamquin** [comment]
Note, "manifest" has referred to a list of goods to be shipped in English since at least 1700 and probably earlier.  Manifestation and manifest have been different English words since at least the 16th century even though they both came to us (through separate paths, in one case via Old French) from the same Latin root. However, if at least one person reported being confused perhaps others would be. I'm still not very convinced I'm afraid but it's for sure up to the editors - thanks.

**Comment by TzviyaSiegman** [comment]
I agree with @prototypo. Further, I think we should remove non-technical uses of the word "manifest(ation)". It would be a good idea to link to the definition of FRBR manifestation for clarity.

**Comment by mattgarrish** [comment]
In one of the darker corners of EPUB you'll find that we [defined and used manifestation in 3.0](http://www.idpf.org/epub/30/spec/epub30-publications.html#gloss-manifestation) (not explicitly linked to FRBR, but with the same general meaning). It was subsequently dumped unceremoniously in 3.0.1 as it was found to cause more confusion than it solved.

As these documents are already published notes, I'm not pushing for changes, but avoiding a repeat of the epub experience as we move forward would wise. I'd go further and try to find alternative wording to "manifestation" whenever possible.

**Comment by liamquin** [comment]
I admit that even though I was reluctant, I asked David to file the issue in the end, and suggested text for the note (if one person is confused it's likely others are, even if I find it very clear) and I'm fine with @TzviyaSiegman 's proposed resolution. It's interesting to see what happens when people come to a document from different backgrounds. But I don't know if or when these documents will be republished.

**Comment by iherman** [comment]
@liamquin indeed it is unclear whether this note will be republished. However, these issues will be relevant for whatever the WG does. Ie, it is good to have them here. Thanks for getting this rolling!
